### ISSUE TITLE
Code update

### ISSUE THREAD

**Issue by qkrtnskfk23** [rheasukthanker/AttnFlow#1]
Hello!

I read your cvpr paper "Generative Flows with Invertible Attentions" and it is really impressive work for developing normalizing flow.

Then, i am trying to study and implement your work, so now i hope that you update code. 

May I know when it will be uploaded? or if uploading is difficult, can i get your code by e-mail?

Thank you.

Best regard.

**Comment by zhiwu-huang** [comment]
Hi,

Sorry for the late reply. The raw code, data and pre-trained models are available at https://zhiwu-huang.github.io/publication.html

Best regards,
Zhiwu

**Comment by WongJaylen** [comment]
> Hi,
> 
> Sorry for the late reply. The raw code, data and pre-trained models are available at https://zhiwu-huang.github.io/publication.html
> 
> Best regards, Zhiwu

Hi,
I can't find the code according to the page you provided

**Comment by cviviers** [comment]
Any update on this? I am also not able to find the code anywhere.

**Comment by zhiwu-huang** [comment]
> Any update on this? I am also not able to find the code anywhere.

As shown on the provided page [here](https://zhiwu-huang.github.io/publication.html) (copied below), the raw code and models are available _upon request_. Please email me at Zhiwu.Huang@soton.ac.uk, and I will grant you access.

_- Rhea Sanjay Sukthanker, Zhiwu Huang, Suryansh Kumar, Radu Timofte, Luc Van Gool. Generative Flows with Invertible Attentions. In Computer Vision and Pattern Recognition (CVPR), 2022. [Preprint ](https://arxiv.org/abs/arXiv:2106.03959)| Code (raw, Size:32G including code, data and pretrained models) [On Request]_

**Comment by cviviers** [comment]
Dear @zhiwu-huang 

Thanks for the response. I have sent you an email with a request for the code.

Looking forward to hearing from you.

Kind regards,
Christiaan
